receiver/awsxrayreceiver/internal/translator/sdk.go

```diff
@@ -0,0 +1,55 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translator
+
+import (
+	"strings"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/translator/conventions"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/common/awsxray"
+)
+
+func populateInstrumentationLibrary(seg *awsxray.Segment, ils *pdata.InstrumentationLibrarySpans) {
+	if seg.AWS != nil && seg.AWS.XRay != nil {
+		xr := seg.AWS.XRay
+		il := ils.InstrumentationLibrary()
```

Comment:
We don't use instrumentation library in exporter, and even if we did it would be for something different than the SDK version - x-ray doesn't propagate something semantically similar to instrumentation library. So let's leave this out.